scripts/commands/readme.js

```diff
@@ -18,12 +17,11 @@ async function updateReadMe( currentBranchName ) {
 	process.chdir( `${ __dirname }/..` );
 
 	if ( ! fs.existsSync( 'readme.txt' ) ) {
-		error(
-			'The file readme.txt does not exist. Verify you are in the wc-calypso-bridge project directory.'
+		return abortAndSwitchToBranch(
+			'The file readme.txt does not exist. Verify you are in the wc-calypso-bridge project directory.',
+			'error',
```

Comment:
Might it be worth using constants for the `error`|`warning`|`info` strings?
```suggestion
			NOTICE_LEVEL_ERROR,
```